public/.htaccess

```diff
@@ -1,5 +1,5 @@
 # Content Security Policy-Headers
 # you have to enable apache module headers to get them working
-#Header set Content-Security-Policy "default-src 'self';"
-#Header set X-Content-Security-Policy "default-src 'self';"
-#Header set X-Webkit-CSP "default-src 'self';"
+#Header set Content-Security-Policy "default-src 'self'; referrer no-referrer;"
+#Header set X-Content-Security-Policy "default-src 'self'; referrer no-referrer;"
+#Header set X-Webkit-CSP "default-src 'self'; referrer no-referrer;"
```

Comment:
Isn't [Referrer Policy Draft](https://w3c.github.io/webappsec-referrer-policy/#referrer-policy-header) declaring a separate header `Referrer-Policy`? I guess it should be a separate header `Referrer-Policy: no-referrer;`.